plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestOraclePoolRemarksReportingConnectorSmokeTest.java

```diff
@@ -110,6 +112,79 @@ private static String varcharLiteral(String value)
         return "'" + value.replace("'", "''") + "'";
     }
 
+    @Test(dataProvider = "testColumnNameDataProvider")
```

Comment:
why is the override necessary? Because instead of COMMENT ON COLUMN failure the CREATE TABLE fails?
Can we instead add a method which filters the dataprovider and override where needed?

Reply:
This is not "override" as it's smoke tests. We need to enable `oracle.remarks-reporting.enabled` for exposing column comments.

Reply:
It doesn't inherit from common class, I see.